releasenotes/notes/python-2-7-17-abbfe429ee77680b.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+other:
+  - |
+    Bump embedded Python to 2.7.17.
```

Comment:
Again, this file is already in master with the same contents, sad that the github diff is wrong :(